### PR TITLE
[15.0][FIX] crm_multicompany_reporting_currency: fix amount

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by yankinmax** [comment]
@gurneyalex can you pls help to get it merged?

File: crm_multicompany_reporting_currency/models/crm_lead.py [hunk 1]
```diff
@@ -79,5 +79,6 @@ def _compute_amount_multicompany_reporting_currency(self):
             if record.company_currency == record.multicompany_reporting_currency_id:
                 to_amount = record.expected_revenue
             else:
-                to_amount = record.expected_revenue / record.currency_rate
+                inverse_rate = 1.0 / record.currency_rate
+                to_amount = record.expected_revenue / inverse_rate
```
Comment (SilvioC2C) [inline crm_multicompany_reporting_currency/models/crm_lead.py]: Mathematically, to divide by the inverse rate is equivalent to multiplying by the original rate, so this snippet of code is equivalent to:
```suggestion
                to_amount = record.expected_revenue * record.currency_rate
```
which (I believe) is easier to read

**Review by SilvioC2C — APPROVED** [review]
Small suggestion, else LGTM

File: crm_multicompany_reporting_currency/models/crm_lead.py (hunk 1 above)
Comment (yankinmax) [reply crm_multicompany_reporting_currency/models/crm_lead.py]: I'll check @SilvioC2C, thanks

**Comment by yankinmax** [comment]
NOTE: pre-commit is failing because of an error caused by pre-commit initialisation itself. So, it's not related to a module.

**Comment by simahawk** [comment]
https://github.com/OCA/sale-reporting/pull/209#issuecomment-1412177358

And shorten the msg :pray: ?
You can keep the explanation for the description of the commit (that serves as a description for the PR too).

**Comment by simahawk** [comment]
> /ocabot merge nobump

@yankinmax you should use `patch` in this case. Although you cannot merge if you are not a maintainer in the target branch.

/ocabot merge patch

**Pull request merged**